config/development/index.js

```diff
@@ -4,7 +4,7 @@ export default {
   dashboardUrl: '',
```

Comment:
shouldn't we also add our Syncano dashboard URL here? like in `syncanoAPIBaseUrl`